runtime/include/chpl-tasks-prvdata.h

```diff
@@ -23,7 +23,14 @@
 // This header file provides chpl_comm_taskPrvData_t
 #include "chpl-comm-task-decls.h"
 
-// The type for task private data
+// Type for Chapel-managed task private data
+// to be copied to new tasks.
+// Includes 'serial_state'
+typedef struct {
+  unsigned char data[32];
```

Comment:
Ah!  I'd looked at that file but rather rapidly just for context and missed the assert().